test/test_add_module.py

```diff
@@ -52,6 +52,9 @@ class TestCode(object):
     OUTCOME_SUCCESS_MESSAGE = "alert('Module %s has been added successfully!');"
```

Comment:
`bad indentation` from this file is from the commented non-functional functions.